scripts/instrumentationRunner.py

```diff
@@ -0,0 +1,58 @@
+"""
+Submits the apks to BrowserStack to run the instrumentation tests.
```

Comment:
since it's only related to browserstack we should rename this something similar to `browserstack.py` to avoid confusion.